### ISSUE TITLE
(t) WARNING - update fails - needs cli follow-up "zypper in -f rockstor"

### ISSUE THREAD

**Issue by phillxnet** [rockstor/rockstor-core#2463]
Current testing channel: Due to the number of changes in our build system our Web-UI update fails when upgrading past 4.5.0. Clean installs, however, works as expected. This issue thus affects only the now EOL Leap 15.3 installs.

## Work around (as root user):
```
zypper in -f rockstor
reboot
```
Note that the *reboot* command may be substituted by *systemctl start rockstor-bootstrap.service*.

## Indicators of this issue
The output of:
```
systemctl status rockstor-pre.service
```
containing:
```
Jan 12 15:58:02 rleap15-3 systemd[1]: Starting Tasks required prior to starting Rockstor...
Jan 12 15:58:03 rleap15-3 poetry[661]: Traceback (most recent call last):
Jan 12 15:58:03 rleap15-3 poetry[661]:   File "<string>", line 1, in <module>
Jan 12 15:58:03 rleap15-3 poetry[661]:   File "/usr/lib64/python2.7/importlib/__init__.py", line 37, in import_module
Jan 12 15:58:03 rleap15-3 poetry[661]:     __import__(name)
Jan 12 15:58:03 rleap15-3 poetry[661]: ImportError: No module named scripts.initrock
```
Due to an incomplete execution in the %post scriptlet re build.sh:
indicated by the following contents in the poetry-install.txt file:
```
cat /opt/rockstor/poetry-install.txt 
./build.sh: line 23: poetry: command not found
```

**Comment by phillxnet** [comment]
Awaiting confirmation that testing channel no longer displays this failure, for the stated (or other) reason.

**Comment by phillxnet** [comment]
Closing as confirmed (for now) as having been resolved by the associated pr and an rpm scriptlet modification that similarly surfaced as a failure of this sort: i.e. incomplete install requiring a zypper forced install (re-install) to fix.

Ergo; closing as:
Fixed by #2464
